src/Microsoft.ML.Data/EntryPoints/TransformModel.cs

```diff
@@ -48,6 +48,16 @@ public sealed class TransformModel : ITransformModel
         /// </summary>
         public ISchema OutputSchema => _chain.Schema;
 
+        /// <summary>
+        /// This contains the transforms to save instantiated on an IDataView with
+        /// appropriate initial schema. Note that the "root" of this is typically either
+        /// an empty IDataView or a BinaryLoader with no rows. However, other root
+        /// types are possible, since we don't insist on this when loading a model
+        /// from a zip file. However, whenever we save, we force a BinaryLoader to
+        /// be serialized for the root.
+        /// </summary>
+        public IDataView View => _chain;
```

Comment:
Let's not do this. I'd feel better about this if instead we used `Apply` on an `EmptyDataView` based on `InputSchema`.